CRM/Member/BAO/Membership.php

```diff
@@ -2532,14 +2522,13 @@ public static function updateMembershipStatus($deceasedParams, $contactType) {
           'source_record_id' => $dao->id,
           'activity_type_id' => array_search('Change Membership Status', $activityTypes),
           'status_id' => 2,
-          'version' => 3,
           'priority_id' => 2,
           'activity_date_time' => CRM_Utils_Time::date('Y-m-d H:i:s'),
           'is_auto' => 0,
           'is_current_revision' => 1,
           'is_deleted' => 0,
         ];
-        civicrm_api('activity', 'create', $activityParam);
+        civicrm_api3('activity', 'create', $activityParam);
```

Comment:
OK - I guess you think it would be better to throw an exception than quietly ignore i this fails - seems reasonably altough  it might flush something out

Reply:
@eileenmcnaughton Not sure I understand this comment, I just changed this one so it used a non-deprecated function.